tests/components/alphaess/test_config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Test the AlphaEss config flow."""
```

Comment:
Since you only have tests for the config flow, you need to exclude the others in .codecoveragerc